plugins/chunter-resources/src/components/ChannelPresenter.svelte

```diff
@@ -13,13 +13,13 @@
 // limitations under the License.
 -->
 <script lang="ts">
-  import type { Channel } from '@anticrm/chunter'
+  import type { ChunterSpace } from '@anticrm/chunter'
```

Comment:
We should have different presenters for channel and DM, because name in DM is empty